src/ert/config/ert_config.py

```diff
@@ -372,12 +372,32 @@ def read_site_config(cls) -> ConfigDict:
 
     @classmethod
     def read_user_config(cls, user_config_file: str) -> ConfigDict:
-        site_config = cls.read_site_config()
-        return lark_parse(
-            file=user_config_file,
-            schema=init_user_config_schema(),
-            site_config=site_config,
-        )
+        return lark_parse(user_config_file, schema=init_user_config_schema())
+
+    @classmethod
+    def read_user_config_and_apply_site_config(
+        cls, user_config_file: str
+    ) -> ConfigDict:
+        site_config_dict = cls.read_site_config()
+        user_config_dict = cls.read_user_config(user_config_file)
+
+        for keyword, value in site_config_dict.items():
+            if keyword == "QUEUE_OPTION":
+                filtered_queue_options = []
+                for queue_option in value:
+                    option_name = queue_option[1]
```

Comment:
I am not a fan of this. Maybe we can change QUEUE_OPTIONS to be `Dict[str, Dict[str, str]` instead of `List[List[str]]`?
Something like ```{
"LSF": {"bsub_cmd": "/path/to/cmd"}
}```

We do this somewhere later on, maybe it can be moved up here instead?
I assume this is due to legacy reasons @berland

Reply:
It seems this would require a larger refactoring of ConfigDict, which will not be of priority right now 🌧️

Reply:
They are List[List[str]] initially so we can reason about the order of settings before we interpret them into a Dict.